firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/FirebaseInAppMessaging.java

```diff
@@ -109,6 +110,42 @@ public boolean isAutomaticDataCollectionEnabled() {
     return dataCollectionHelper.isAutomaticDataCollectionEnabled();
   }
 
+  /**
+   * Enable, disable or clear automatic data collection for Firebase In-App Messaging.
+   *
+   * <p>When enabled, generates a registration token on app startup if there is no valid one and
```

Comment:
@VinayGuthal 
Just confirming that unclosed <p> tags will get processed correctly?

Reply:
yes as per https://stackoverflow.com/questions/33787753/javadoc-closing-tags